src/frontend/frontend.jl

```diff
@@ -140,10 +140,10 @@ const ps_load_columns = Dict( # This is meant for our use, not for importing pow
 :susceptance -> line susceptance
 :ang_min -> minimum voltage angle difference
 :ang_max -> maximum voltage angle difference
-:q_to -> Reactive power reaching `to_bus`
-:q_from -> Reactive power leaving `from_bus`
-:p_to -> Real power reaching `to_bus`
-:p_from -> Real power leaving `from_bus`
+:g_to ->
+:g_fr ->
+:b_to ->
+:b_fr ->
```

Comment:
Why remove the documentation here?

Reply:
I was not completely sure what these entries are about (contextually, their units). I'm on it.

Reply:
This is done.